docs/src/main/paradox/docs/releases/v1.7-to-v1.8-migration.md

```diff
@@ -0,0 +1,447 @@
+    * Count the indexed documents in Blazegraph views across the two instances
+    * Results are written in `public.migration_blazegraph_count`
+* `/migration/composite`
+    * Count the indexed documents in composite views across the two instances
+    * Results are written in `public.migration_composite_count`
+* `/migration/resources`
+    * Results are written in `public.migration_resources_diff`
+
+The calls must be authenticated using a service account.
+
+Before running the tests, the following command needs to be run in your PostgreSQL instance:
+
+```sql
+CREATE TABLE IF NOT EXISTS public.migration_resources_diff_offset(
+    project   text         NOT NULL,
+    ordering  bigint,
+    PRIMARY KEY(project)
+);
+
+CREATE TABLE IF NOT EXISTS public.migration_resources_diff(
+    type      text         NOT NULL,
+    project   text         NOT NULL,
+    id        text         NOT NULL,
+    value_1_7 JSONB,
+    value_1_8 JSONB,
+    error     text,
+    PRIMARY KEY(type, project, id)
+);
+
+CREATE TABLE IF NOT EXISTS public.migration_project_count(
+    project   text         NOT NULL,
+    event_count_1_7 bigint,
+    event_count_1_8 bigint,
+    resource_count_1_7 bigint,
+    resource_count_1_8 bigint,
+    error     text,
+    PRIMARY KEY(project)
+);
+
+CREATE TABLE IF NOT EXISTS public.migration_blazegraph_count(
+    project   text         NOT NULL,
+    id        text         NOT NULL,
+    count_1_7 bigint,
+    count_1_8 bigint,
+    PRIMARY KEY(project, id)
+);
+
+CREATE TABLE IF NOT EXISTS public.migration_elasticsearch_count(
+    project   text         NOT NULL,
+    id        text         NOT NULL,
+    count_1_7 bigint,
+    count_1_8 bigint,
+    PRIMARY KEY(project, id)
+);
+
+CREATE TABLE IF NOT EXISTS public.migration_composite_count(
+    project       text         NOT NULL,
+    id            text         NOT NULL,
+    space_id text         NOT NULL,
+    count_1_7 bigint,
+    count_1_8 bigint,
+    PRIMARY KEY(project, id, space_id)
+);
+```
+
+#### Example
+
+```shell
+export TOKEN=<service-account-bearer-token>
+curl --request POST -H "Authorization: Bearer $TOKEN" http://localhost:80/v1/migration/elasticsearch
+```
+
+In the logs, you should observe
+
+```text
+2023-02-27 09:11:37 INFO  c.e.b.n.d.p.c.m.ElasticSearchViewsCheck - Starting checking elasticsearch views counts
+...
+2023-02-27 09:11:39 INFO  c.e.b.n.d.p.c.m.ElasticSearchViewsCheck - Checking elasticsearch views counts completed in 1 seconds.
+2023-02-27 09:11:39 INFO  c.e.b.n.d.p.c.m.MigrationCheckRoutes - Stopping stream 'elasticsearch' after completion.
+```
+
+After which it is possible to inspect the results in the `public.migration_elasticsearch_count` PostgreSQL table.
+
+## Indexing
+
+Once migration is completed and the previous checks have been performed, Delta can be restarted in normal mode.
+
+* Shut down Delta v1.8 running in migration mode.
+* Remove the `MIGRATE_DATA`, `DISABLE_INDEXING` environment variables.
```

Comment:
We should indicate that the different indexing processes can be done independently